sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Listeners/EventHubsTargetScaler.cs

```diff
@@ -86,7 +86,7 @@ internal static TargetScalerResult ThrottleScaleDownIfNecessaryInternal(TargetSc
         internal TargetScalerResult GetScaleResultInternal(TargetScalerContext context, long eventCount, int partitionCount)
         {
             int desiredConcurrency = GetDesiredConcurrencyInternal(context);
-            int desiredWorkerCount = (int)Math.Ceiling(eventCount / (decimal)desiredConcurrency);
+            long desiredWorkerCount = (long)Math.Ceiling(eventCount / (decimal)desiredConcurrency);
```

Comment:
should we think about enforcing a max on the desired worker count == int.max

Reply:
in this case, we are effectively doing the same thing 

the largest target worker count for event hub is the last element in the sortedValidWorkerCountList (which is the partition count) -- see `GetValidWorkerCount`

Reply:
whats the advantage of doing it this way? Its a bit harder to understand than doing a math.max

Reply:
logically there's no difference, I suppose to enforce the max approach, we'd have to save an extra variable long desiredWorkerCount to compare against maxInt. We'd then use this variable to pass into `GetValidWorkerCount`